libs/carrier-portal/member-coverage/src/lib/member-coverage/person-contact-info.component.html

```diff
@@ -0,0 +1,25 @@
+<ng-template [ngIf]="hasContactInfo()">
+  <h2>Contact Info</h2>
+  <dl>
+    <ng-template [ngIf]="person.home_address">
```

Comment:
Not a huge deal, but generally the ng-template would be used for the parent, but these children would ng-content. functionally works the same, but figured i would point out